src/aria/utils/Orientation.js

```diff
@@ -33,6 +37,8 @@ module.exports = Aria.classDefinition({
      * Adding a listener while initializing Orientation to listen to the native orientationchange event.
      */
     $constructor : function () {
+        this.$logWarn("This class is deprecated and will be removed from the framework, please use aria.utils.Device instead");
+
```

Comment:
- Backward compatibility: sure
- Warning: this is on purpose as $logWarn already indicates the class name
![warning](https://cloud.githubusercontent.com/assets/1153034/5774215/cd6fac36-9d6b-11e4-8565-b65e3c6497c8.gif)